test/chruby_auto_test.sh

```diff
@@ -121,6 +121,12 @@ function test_chruby_auto_invalid_ruby_version()
 		     "$expected_auto_version" "$RUBY_AUTO_VERSION"
 }
 
+function test_chruby_auto_preserves_dollar_underscore()
+{
+	: expected
```

Comment:
Might want to be

```shell
: expected 1 2 3
assertEquals "value of $_ does not change" "$_" "expected 1 2 3"
```

Reply:
The `:` is just a dummy command - one that returns true and has the side-effect of setting `$_`.

Reply:
Could this be rewritten as explicit code that sets `$_` or returns an a return value?